### PR TITLE
README fixes

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -145,3 +147,11 @@ simply be appended to the invocation as follows::
...
 
 
+Limitations
+-----------
+This package does not support the fluent `Cursor API <https://api.mongodb.com/python/current/api/pymongo/cursor.html>`_,
+so if you attempt to use it like so::
+
+    ExplainCollection(collection).find({}).sort(...)
+
+it will fail.
```
Comment (ShaneHarvey) [inline README.rst:157]: Can you offer a suggested workaround? Something like: 
```
Instead pass all the arguments to the find() call, like so::

    ExplainCollection(collection).find({}, sort=...)

```

File: README.rst [hunk 2]
```diff
@@ -134,9 +136,9 @@ within the specified script, **in addition to running every command** in the scr
 explain output is generated using the `logging <https://docs.python.org/3/library/logging.html>`_ module,
 if your script configures logging module there are certain things to keep in mind:
 
-- if your script sets the `logging level <https://docs.python.org/3/library/logging.html#logging-levels>`_
-    higher than INFO, the explain output will be suppressed entirely.
-- the explain output will be sent to whatever stream your script configures the logging module to send output to.
+    - if your script sets the `logging level <https://docs.python.org/3/library/logging.html#logging-levels>`_
+        higher than INFO, the explain output will be suppressed entirely.
+    - the explain output will be sent to whatever stream your script configures the logging module to send output to.
```
Comment (ShaneHarvey) [inline README.rst:141]: This still doesn't look correct in the rendered view: https://github.com/mongodb-labs/pymongoexplain/blob/b4bd835c930b496f0641a6a81f511244a21c179e/README.rst#explaining-commands-in-a-script

File: README.rst [hunk 3]
```diff
@@ -145,3 +147,11 @@ simply be appended to the invocation as follows::
     python3 -m pymongoexplain <path/to/your/script.py> [PARAMS] [--optname OPTS]
 
 
+Limitations
+-----------
+This package does not support the fluent `Cursor API <https://api.mongodb.com/python/current/api/pymongo/cursor.html>`_,
```
Comment (prashantmital) [inline README.rst:152]: This is not the correct link for PyMongo documentation. It should be: https://pymongo.readthedocs.io/en/stable/api/pymongo/cursor.html

File: README.rst [hunk 4]
```diff
@@ -38,7 +38,7 @@ To install this package simply use pip::
 Support / Feedback
 ==================
 
-For questions, discussions, or general technical support, visit the MongoDB Community Forums.
+For questions, discussions, or general technical support, visit the `MongoDB Community Forums <https://developer.mongodb.com/community/forums/tag/python>`_.
```
Comment (prashantmital) [inline README.rst:41]: Please add a link to this GitHub repository in the reference to 'this GitHub repository' on L48

File: README.rst [hunk 5]
```diff
@@ -145,3 +147,11 @@ simply be appended to the invocation as follows::
     python3 -m pymongoexplain <path/to/your/script.py> [PARAMS] [--optname OPTS]
 
 
+Limitations
+-----------
```
Comment (prashantmital) [inline README.rst:151]: Please leave a blank line after the title.

File: README.rst (hunk 1 above)
Comment (juliusgeo) [reply README.rst:157]: Done.

File: README.rst (hunk 4 above)
Comment (juliusgeo) [reply README.rst:41]: Done.

File: README.rst (hunk 2 above)
Comment (juliusgeo) [reply README.rst:141]: Done.

File: README.rst (hunk 5 above)
Comment (juliusgeo) [reply README.rst:151]: Done.

File: README.rst (hunk 3 above)
Comment (juliusgeo) [reply README.rst:152]: Done.

**Pull request merged**